cmd/coreinstance/create_core_instance_k8s.go

```diff
@@ -166,3 +182,14 @@ func newCmdCreateCoreInstanceOnK8s(config *cfg.Config, testClientSet kubernetes.
 
 	return cmd
 }
+
+// GetK8sYaml strips empty properties which would typically be marshalled with yaml.Marshal
+func GetK8sYaml(req interface{}) {
```

Comment:
If return object from k8s is marshalled into Yaml all empty fields will be marshalled as well. Only with JSON that doesn't happen. 
So this is a clumsy workaround that.